deploy/vault-secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: keyring-secret-vault
+type: Opaque
+stringData:
+  keyring_vault_n1.conf: |-
+    token = token
+    vault_url = http://1.1.1.1:8200
+    secret_mount_point = secret
```

Comment:
As I just discovered, this config doesn't work.

keyring_vault requires a non empty `vault_ca` parameter even when using HTTP (not HTTPS), otherwise it silently won't save any keys in the vault.